test/playground.html

```diff
@@ -2,7 +2,68 @@
 <html lang="en">
 	<title>O hai</title>
 
-	<div class="foo" id="foo">foo</div>
+	<div class="listbox-area">
```

Comment:
Glad I'm not the only one who submits changes to the playground file :)

Reply:
whahaha, fair enough :P